Makefile

```diff
@@ -40,48 +31,25 @@ $(NIS) setup: package.json
 lint: $(NIS)
 	$(NPM) run lint
 
-build/katex.js: katex.js $(wildcard src/*.js) $(NIS)
-	mkdir -p build
-	$(BROWSERIFY) -t [ babelify ] $< --standalone katex > $@
-
-build/katex.min.js: build/katex.js
-	mkdir -p build
-	$(UGLIFYJS) < $< > $@
-
-build/katex.css: static/katex.less $(wildcard static/*.less) $(NIS)
-	mkdir -p build
-	./node_modules/.bin/lessc $< $@
-
-build/katex.min.css: build/katex.css
+webpack: katex.js $(wildcard src/*.js) $(wildcard static/*.less) $(NIS)
 	mkdir -p build
-	$(CLEANCSS) -o $@ $<
+	mkdir -p build/contrib
+	mkdir -p build/fonts
+	@# Since everything in build/contrib and build/fonts is put in the built files,
+	@# make sure there's nothing in there we don't want.
+	rm -rf build/contrib/*
+	rm -rf build/fonts/*
```

Comment:
I don't if we should just do `rm -rf build`?

Reply:
@kevinbarabash I think preserving other files in `build/` is no longer necessary as dev server doesn't generate or use files in `build/`. Will change to `rm -rf build/*`